### ISSUE TITLE
Installation cannot find gem Mashal4.8/JWt 1.5.3

### ISSUE THREAD

**Issue by michael-mohemian** [fastlane/fastlane#3923]
Run "sudo gem install supply"

Output:
ERROR:  While executing gem ... (Gem::RemoteFetcher::FetchError)
    bad response Not Found 404 (http://rubygems.global.ssl.fastly.net/quick/Marshal.4.8/jwt-1.5.3.gemspec.rz)

**Comment by asfalcone** [comment]
@michael-mohemian thanks for the report. `jwt`

Can you try doing `sudo gem install jwt` then `sudo gem install supply` again?

It looks like `jwt` 1.5.3 was possibly yanked from rubygems since the highest available `jwt` gem is 1.5.2 right now https://rubygems.org/gems/jwt 

Let me know if that helps! :+1:

**Comment by alexkrzyzanowski** [comment]
```sh
MacBook-Pro-Aleksandr:kisskiss-ios aperechnev$ sudo gem install jwt
ERROR:  While executing gem ... (Gem::RemoteFetcher::FetchError)
    bad response Not Found 404 (https://rubygems.global.ssl.fastly.net/quick/Marshal.4.8/jwt-1.5.3.gemspec.rz)
MacBook-Pro-Aleksandr:kisskiss-ios aperechnev$
```

**Comment by asfalcone** [comment]
@alexkrzyzanowski  `sudo gem install jwt jwt-1.5.2` to force 1.5.2 ?

**Comment by alexkrzyzanowski** [comment]
@asfalcone possibly yes, but I can't install *fastlane* because it tries to install _jwt-1.5.3_.

**Comment by pkester** [comment]
Supply still tries to remote fetch https://rubygems.global.ssl.fastly.net/quick/Marshal.4.8/jwt-1.5.3.gemspec.rz no matter what version you have installed. I installed the dev version of 1.5.4 using the gem spec and get the same result. Is there a way to skip supply as my project doesn't have android support anyhow?

**Comment by asfalcone** [comment]
I'm not seeing the behavior as described. I uninstalled `jwt` 1.5.3 and then installed 1.5.2 directly. I can then install fastlane and supply without issue. fastlane and supply don't have direct dependencies on jwt, and our transitive dependencies only need ~> 1.5

Please give this a try
```
gem uninstall jwt
gem install jwt -v 1.5.2 --verbose
```

Please see this thread here as well about jwt 1.5.3 getting yanked: https://github.com/jwt/ruby-jwt/issues/133

**Comment by excpt** [comment]
Sorry for the trouble the ruby jwt 1.5.3 version caused. I just released version 1.5.4.

**Comment by pkester** [comment]
I uninstalled all versions of jwt gems on my primary machine and it worked. I also ran a fresh install of fastlane on my dev laptop and can confirm everything works fine now with fastlane installation and all of its dependencies.

Thanks everyone for the quick response times.

**Comment by asfalcone** [comment]
:+1: Going to close this since there is resolution in both using an older version and using the newly released 1.5.4.
